<?xml version="1.0"?>
<launch>

  <arg name="camera_id"       default="SwissRanger" />
  <arg name="scale_factor"    default="4" />
  <arg name="camera_info_url" default="file://$(env HOME)/.ros/camera_info/$(arg camera_id).yaml" />
  
  <node pkg="swissranger_camera" type="image_publisher_sr" name="$(arg camera_id)_image_publisher" output="screen">
    <param name="scale_factor"        value="$(arg scale_factor)" />
    <param name="camera_info_url"     value="$(arg camera_info_url)" />
    <param name="camera_name"         value="$(arg camera_id)" />
    
    <param name="image_resized_topic" value="/$(arg camera_id)/intensity/image_resized" />
    <param name="camera_info_topic"   value="/$(arg camera_id)/intensity/camera_info" />
    <param name="pointcloud_topic"    value="/$(arg camera_id)/pointcloud2_raw" />
  </node>
  
</launch>
